Csv export

ciabhanconnelly opened the pull request:
Added csv (well, tsv) export for you to review. Sorry for the previous mistakes, not too familiar with github for big projects like this yet.

ciabhanconnelly committed:
Fixed errors

Made some dumb github mistakes, fixing things on this branch

ciabhanconnelly committed:
Blank entries

Fixed an issue where an error would sometimes occur when the user did not enter a date. Now all reports will always be selected.

harshilshah4251 committed:
resolving conflict

ciabhanconnelly committed:
Config file with button

ciabhanconnelly committed:
Removed console.log

ciabhanconnelly committed:
Fixed Date Range

ciabhanconnelly committed:
Merge branch 'csv-export' of https://github.com/TID-Lab/aggie into csv-export

limmabean commented:
Nothing happens. I click to adjust the times and I get the following codes: GET /api/v1/viz/?before=2021-02-28T16:08:00.277Z&after=2019-09-04T16:08:00.277Z 200 3917.541 ms - 52447548
GET /api/v1/viz/?before=2021-02-28T16:08:00.277Z&after=2019-09-04T16:08:00.277Z 304 3263.679 ms - -

ciabhanconnelly committed:
Improvements

Let's see if this works on other devices. Working better on mine over large numbers of reports... though mongodump is still better if you really need to do a lot...

limmabean reviewed (approved):
Works for me. Good job!

limmabean commented:
Remove package-lock.json in the next commit.

ciabhanconnelly committed:
Update package-lock.json

ciabhanconnelly committed:
Removing Package Lock Edits

ciabhanconnelly committed:
Removing Package Lock Edits

ciabhanconnelly committed:
Merge branch 'develop' into csv-export
